lib/jnpr/jsnapy/snap.py

```diff
@@ -135,19 +135,21 @@ def run_cmd(self, test_file, t, formats, dev, output_file, hostname, db):
         self.logger_snap.debug(colorama.Fore.BLUE +
                                "Tests Included: %s " %t,
                                extra=self.log_detail)
-        self.logger_snap.info(
-            colorama.Fore.BLUE +
-            "Taking snapshot of COMMAND: %s " %
-            command,
-            extra=self.log_detail)
+        if command not in command_list:
```

Comment:
This is not the correct way to implement unique command execution. This only stops the printing. In the `else` scenario, the snapshot is still taken and the optimization is not at all achieved